main.cpp

```diff
@@ -248,32 +303,41 @@ bool ShoppingCart::rollsDownHill (bool steepHill, bool fast)
         return fast;
     return !fast;
 }
-bool ShoppingCart::isParked (std::string cartLocation)
+void ShoppingCart::isParked(int numCartsInCorral = 1, std::string  cartLocation = "corral", int totalNumCarts = 100)
 {
-    if(cartLocation == "corral") 
-        return true;
-    return false;
+    int missingCarts = totalNumCarts - numCartsInCorral;
+    if(cartLocation == "corral")
+        std::cout << "Cart is safely parked in " << cartLocation << std::endl;
+    std::cout << missingCarts << " cart(s) need(s) to be moved to the cart coarral." << std::endl;
 }
 
 struct Wind
 {
+    Wind();
     double speed = 3.4234898;
     std::string direction = "North West";
     std::string definition = "movement in air molecules";
     std::string cause = "difference in gas density";
     std::string use = "wind energy";
 
-     bool transportSeeds (int numSeed = 4, float gust = 3.42f);
+     void transportSeeds (int, std::string, int);
      bool turnTurbines (std::string, float gust = 34.2f);
      void helpBirdTravel (std::string direction2, bool birdFly, float gust = 34.2f);
 }; 
-bool Wind::transportSeeds (int numSeed, float gust)
+
+Wind::Wind()
 {
-    if(numSeed<=10 && gust > 30)
-        return true;
-    return false;
+    std::cout << "Wind being constructed!" << std::endl;
+}
 
+void Wind::transportSeeds (int numSeeds, std::string treeName, int milesTraveled)
+{
+    if(treeName == "poplar")
+        numSeeds = 25000000;
+        milesTraveled = 20;
+        std::cout << "Strong wind can disperse " << numSeeds << " " << treeName << " seeds up to " << milesTraveled << " miles." << std::endl;
 }
```

Comment:
Fix this logic.
Be explicit with your `{ }` to indicate what code should be executed when `treeName == poplar`
- [ ] revise